### PR TITLE
Feature/hw 66767 v4 authentication token

### PR DESCRIPTION
**Pull request by Ssangaran**

### CODE REVIEW

**Review by Rrathinasabapath — APPROVED** [review]
Verified the changes. They are ready to be reviewed by Vadim.

File: src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java [hunk 1]
```diff
@@ -52,7 +52,6 @@ public Hyperwallet(final String username, final String password, final String pr
     public Hyperwallet(final String username, final String password, final String programToken, final String server) {
         this(username, password, programToken, server, null);
     }
-
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java:55]: Revert this file change

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletAuthenticationToken.java [hunk 2]
```diff
@@ -10,6 +10,10 @@
 
     private String value;
 
+    /**
```
Comment (ramahalingam) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletAuthenticationToken.java:13]: Java doc not required for pojo class

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 3]
```diff
@@ -387,6 +386,7 @@ public void testGetUserStatusTransition_noTransitionToken() {
         }
     }
 
+
```
Comment (ramahalingam) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java:389]: Revert the file changes.

**Review by ramahalingam — COMMENTED** [review]
Please write an IT test in the file : HyperwalletIT.java

**Review by vkorotchenko — APPROVED** [review]

**Pull request merged**